TriblerGUI/widgets/trustpage/js/help_page.js

```diff
@@ -0,0 +1,329 @@
+function showHelpPage () {
+    document.getElementById("help-button").innerHTML = "Close";
+    document.getElementById("help-overlay").style.display = "block";
+    document.getElementById("help-page").style.display = "block";
+    document.getElementById("graph").style.opacity = "0.5";
+    drawSvgs();
+}
+
+function closeHelpPage () {
+    document.getElementById("help-button").innerHTML = "Help";
+    document.getElementById("help-overlay").style.display = 'none';
+    document.getElementById("help-page").style.display = 'none';
+    document.getElementById("graph").style.opacity = "1";
+}
+
+function drawSvgs () {
+    drawIdentitySvg();
+    drawReputationSvg();
+    drawNodeSizeSvg();
+    drawConnectionSizeSvg();
+    drawDividingLineSvg();
+}
+
+function drawIdentitySvg () {
```

Comment:
Nice separation of drawing logic, please add as private methods in the new format _drawIdentitfySvg()